Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -109,6 +120,55 @@ - (void)createTransaction:(UIViewController *)controller
     }
 }
 
+- (void)createTimeToDisplay:(UIViewController *)controller
+{
+    SentrySpan *vcSpan = [self viewControllerPerformanceSpan:controller];
+
+    if (![vcSpan isKindOfClass:[SentryTracer self]]) {
+        // Since TTID and TTFD are meant to the whole screen
+        // we will not track child view controllers
+        return;
+    }
+
+    if ([vcSpan.tracer getMiddlewaresOfType:SentryTimeToDisplayTracker.self].count > 0) {
+        // Already tracking time to display.
```

Comment:
`l`: Please describe the situation that could lead to reaching this code path in the comment.

Reply:
By calling loadView more than once. This is not supposed to happen but is possible.

This is just a safe guard.